scipy/stats/_binned_statistic.py

```diff
@@ -598,13 +602,25 @@ def binned_statistic_dd(sample, values, statistic='mean',
             result[vv, a] = flatsum
     elif statistic == 'median':
         result.fill(np.nan)
-        _calc_binned_statistic(Vdim, binnumbers, result, values, np.median)
+        if values[0][0] != None:
+            from ._calc_binned_statistic_pythran import _calc_binned_statistic_pythran
+            _calc_binned_statistic_pythran(Vdim, binnumbers, result, values, statistic)
```

Comment:
This import is repeated a lot here. It looks to me like it should only be done once, at the top of the file. It's fine to do the import even if we don't need it, it's better than repeating it on every function call.